src/edu/stanford/dlss/was/WasapiResponse.java

```diff
@@ -0,0 +1,85 @@
+package edu.stanford.dlss.was;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class WasapiResponse {
+
+  public WasapiResponse() { }
+
+  public ResponseObject parse(byte[] jsonData) throws IOException {
+    ObjectMapper objectMapper = new ObjectMapper();
+    ResponseObject responseObject = objectMapper.readValue(jsonData, ResponseObject.class);
+    return responseObject;
+  }
+
+
+  /**
+   * Class corresponding to JSON received after making WASAPI request ("FileSet")
+   * @see https://github.com/WASAPI-Community/data-transfer-apis/blob/master/ait-implementation/wasapi/implemented-swagger.yaml#L184-L213
+   */
+  public static class ResponseObject {
```

Comment:
what I really wanted to do was have the WasapiResponse be the bean-like object for jackson json parser, but that doesn't work.   So it evolved into an inner static class, because jackson needs inner classes to be static.   If I were to make this inner class a brand new class of its own, then I would want to call it `WasapiResponse` ... so what should the outer class be named?   I'm open to this, just got bogged down in details.